src/components/job-common.js

```diff
@@ -161,3 +161,12 @@ export const workflowDetailsBreadcrumbSubtitle = (namespace, workspaceName, subm
     h3({ style: Style.elements.sectionHeader }, [`Workflow ${workflowId}`]),
   ]);
 };
+
+export const getTaskCost = (vmCost, startTime, endTime) => {
```

Comment:
An options object might be nice here, since there are several arguments to the function.
https://www.codereadability.com/what-are-javascript-options-objects/